policy/release/rpm_repos.rego

```diff
@@ -106,7 +106,9 @@ all_rpm_purls contains purl if {
 	some component in sbom.components
 	purl := component.purl
 
-	# I'm assuming this is faster than parsing it and checking the type
+	# I'm assuming this is faster than parsing it and checking the type.
+	# It would be more correct to match exactly "pkg:rpm/" and "pkg:rpmmod/"
```

Comment:
Easy enough to do something like this if you want to aim for correctness:

```rego
_is_rpmish(purl) if {
  startswith(purl, "pkg:rpm/")
} else if {
  startswith(purl, "pkg:rpmmod/")
}
```

Reply:
Hmm yeah, that's a little tidier that what I had. Let's do it.

Reply:
I added it, works nicely. No other changes, so I'm taking your approval as still good.